src/routes/docs/products/storage/quick-start/+page.markdoc

```diff
@@ -52,74 +57,100 @@ To upload a file, add this to you web, Flutter, Apple, or Android app.
     final storage = Storage(client);
 
     final file = await storage.createFile(
-      bucketId: '[BUCKET_ID]',
+      bucketId: '<BUCKET_ID>',
       fileId: ID.unique(),
       file: InputFile.fromPath(path: './path-to-files/image.jpg', filename: 'image.jpg'),
     );
   }
   ```
+
   ```client-apple
   import Appwrite
 
   func main() async throws {
       let client = Client()
         .setEndpoint("https://cloud.appwrite.io/v1")
-        .setProject("<PROJECT_ID>")
+        .setProject('<PROJECT_ID>')
 
       let storage = Storage(client)
 
       let file = try await storage.createFile(
-          bucketId: "[BUCKET_ID]",
+          bucketId: '<BUCKET_ID>',
           fileId: ID.unique(),
           file: InputFile.fromBuffer(yourByteBuffer,
-              filename: "image.jpg",
-              mimeType: "image/jpeg"
+              filename: 'image.jpg',
+              mimeType: 'image/jpeg'
           )
       )
   }
   ```
+
   ```client-android-kotlin
   import io.appwrite.Client
   import io.appwrite.services.Storage
+  import io.appwrite.models.InputFileio.appwrite.models.InputFile
+  import io.appwrite.ID
 
   suspend fun main() {
       val client = Client(applicationContext)
-          .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
-          .setProject("<PROJECT_ID>") // Your project ID
+          .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+          .setProject('<PROJECT_ID>') // Your project ID
 
       val storage = Storage(client)
 
       val file = storage.createFile(
-          bucketId = "[BUCKET_ID]",
+          bucketId = '<BUCKET_ID>',
           fileId = ID.unique(),
-          file = File("./path-to-files/image.jpg"),
+          file = InputFile.fromPath('./path-to-files/image.jpg'),
       )
   }
   ```
 
   ```http
-  POST /v1/storage/buckets/{bucketId}/files HTTP/1.1
-  Content-Type: multipart/form-data; boundary="cec8e8123c05ba25"
-  Content-Length: *Length of your entity body in bytes*
-  X-Appwrite-Project: [PROJECT_ID]
+  POST https://cloud.appwrite.io/v1/storage/buckets/<BUCKET_ID>/files HTTP/1.1
+  X-Appwrite-Project: <PROJECT_ID>
+  X-Appwrite-Response-Format: 1.5.0
+  Cookie: <COOKIE>
+  Content-Type: multipart/form-data; boundary=---1234
+  Content-Length: <LENGTH>
+  -----1234
+  Content-Disposition: form-data; name='fileId'   
+  <FILEID>
```

Comment:
shouldn’t there be a newline between? Also, let’s add an underscore for consistency.

```suggestion

  <FILE_ID>
```